src/plugins/mce/plugin.c

```diff
@@ -38,10 +38,30 @@ typedef struct _MceData
     NRequest       *request;
     NSinkInterface *iface;
     gchar          *pattern;
+    gboolean        playing;
 } MceData;
 
 static GList *active_events;
 
+static void
+activate_cb (NCore *core, DBusMessage *msg, void *userdata)
+{
+    (void) core;
+    (void) msg;
+    gchar *pattern = (gchar*) userdata;
```

Comment:
nit: As this is C: superfluous void pointer cast. Then again, the already existing code is littered with these... nevermind